File: Modul 4/Python/PRAK402-23108317310005-MuhammadNurwahyudiAdhitama.py
batas = int(input(""))

if batas % 2 != 0:
    batas -= 1

for i in range(1, batas + 1, 2):
    print(i, end=" ")

print()  

for i in range(batas, 1, -2):
    print(i, end=" ")

File: Modul 4/C/PRAK402-23108317310005-MuhammadNurwahyudiAdhitama.c
#include <stdio.h>

int main() {
    int batas, i;

    scanf("%d", &batas);

    if (batas % 2 != 0) {
        batas--;
    }

    for (i = 1; i <= batas; i += 2) {
        printf("%d ", i);
    }

    printf("\n");

    for (i = batas; i >= 2; i -= 2) {
        printf("%d ", i);
    }

    return 0;
}
